server/commands.lua

```diff
@@ -253,16 +253,17 @@ end, 'user')
 -- Me command
 
 QBCore.Commands.Add('me', 'Show local message', {{name = 'message', help = 'Message to respond with'}}, false, function(source, args)
+    if #args < 1 then TriggerClientEvent('QBCore:Notify', source, Lang:t('error.missing_args2'), 'error') return end
     local ped = GetPlayerPed(source)
     local pCoords = GetEntityCoords(ped)
-    local msg = table.concat(args, ' ')
-    if msg == '' then return end
-    if string.match(msg, "<") then TriggerClientEvent('QBCore:Notify', source, Lang:t('error.wrong_format'), 'error') return end
-    for k,v in pairs(QBCore.Functions.GetPlayers()) do
-        local target = GetPlayerPed(v)
+    local msg = table.concat(args, ' '):gsub('[~<].-[>~]', '')
+    local Players = QBCore.Functions.GetPlayers()
+    for i=1, #Players do
+        local Player = Players[i]
+        local target = GetPlayerPed(Player)
```

Comment:
This doesn't work, the GetPlayerPed native accepts the source only, not a player object

Reply:
Nevermind, I now see that that is the source

Reply:
I've labeled it as `Player` referring to the player's source, rather then a player object.